### ISSUE TITLE
Sample app outputs "Failed to create to SampleApplication!" and doesn't start on Raspbian Stretch

### ISSUE THREAD

**Issue by ComputerGuy101** [alexa/avs-device-sdk#1359]
**IMPORTANT**: Before you create an issue, please take a look at our [Issue Reporting Guide](https://github.com/alexa/avs-device-sdk/wiki/Issue-Reporting-Guide).

## Briefly summarize your issue:

When starting the sample app, the terminal outputs a lot of error messages and finishes with "Failed to create to SampleApplication!".

## What is the expected behavior?

The sample app will successfully start and allow me to authorize it.

## What behavior are you observing?

After entering "sudo bash startsample.sh" into the terminal, the sample app begins it's startup but displays what look like a lot of error messages. It then gives the code and link to authorize it but then quickly displays "Failed to create to SampleApplication!" and terminates the application. Here's the debug log: [raspberry_pi_avs_log.txt](https://github.com/alexa/avs-device-sdk/files/3195215/raspberry_pi_avs_log.txt)

## Tell us about your environment:

I am using a Raspberry Pi 2B with a USB WIFI Antenna, a USB microphone, and an analog speaker connected to the audio jack.

### What version of the AVS Device SDK are you using?
      <1.12.1>

### Tell us what hardware you're using:
- [ ] Desktop / Laptop
- [x] Raspberry Pi
- [ ] Other - tell us more:

### Tell us about your OS (Type & version):
- [ ] Linux
- [ ] MacOS 
- [x] Raspbian Stretch
- [ ] Raspbian Jessy
- [ ] Other - tell us more:

**Comment by thardbarger** [comment]
Same problem here. Started after round of upgrades. Am guessing caused by Linux kernel or firmware upgrade.

**Comment by roadcba** [comment]
Same here. No solution yet?

**Comment by ComputerGuy101** [comment]
@roadcba No, nothing yet. @thardbarger might be right that it's just a problem with the newest Raspbian update. This weekend I'll try downloading the sample app on an older version of Raspbian and see if it works.

**Comment by roadcba** [comment]
Yes, same thing happen to me. I have an old card with raspbian jessie installed and the old avs from https://github.com/alexa/alexa-avs-sample-app/wiki/Raspberry-Pi (unavailable now) and works perfectly

**Comment by yuan0258** [comment]
The root cause seems PortAudio problem. 
Maybe you could try to re-install it
==================================
Expression 'alsa_snd_pcm_hw_params_set_period_size_near( pcm, hwParams, &alsaPeriodFrames, &dir )' failed in 'src/hostapi/alsa/pa_linux_alsa.c', line: 923
Expression 'alsa_snd_pcm_hw_params_set_period_size_near( pcm, hwParams, &alsaPeriodFrames, &dir )' failed in 'src/hostapi/alsa/pa_linux_alsa.c', line: 923
Expression 'alsa_snd_pcm_hw_params_set_period_size_near( pcm, hwParams, &alsaPeriodFrames, &dir )' failed in 'src/hostapi/alsa/pa_linux_alsa.c', line: 923
Expression 'paInvalidSampleRate' failed in 'src/hostapi/alsa/pa_linux_alsa.c', line: 2043
Expression 'PaAlsaStreamComponent_InitialConfigure( &self->capture, inParams, self->primeBuffers, hwParamsCapture, &realSr )' failed in 'src/hostapi/alsa/pa_linux_alsa.c', line: 2714
Expression 'PaAlsaStream_Configure( stream, inputParameters, outputParameters, sampleRate, framesPerBuffer, &inputLatency, &outputLatency, &hostBufferSizeMode )' failed in 'src/hostapi/alsa/pa_linux_alsa.c', line: 2838
2019-05-19 14:30:07.786 [  1] C PortAudioMicrophoneWrapper:Failed to open PortAudio default stream:errorCode=-9997
2019-05-19 14:30:07.787 [  1] C PortAudioMicrophoneWrapper:Failed to initialize PortAudioMicrophoneWrapper

**Comment by dhpp** [comment]
Thanks @yuan0258, that does appear to be the problem.  @ComputerGuy101, could you please double check your dependencies per [here](https://github.com/alexa/avs-device-sdk/wiki/Dependencies), and also cross-reference existing tickets that show similar error trace such as #1208, and let us know if you make any progress?

**Comment by ComputerGuy101** [comment]
Sorry I disappeared, I've been very busy. Today I tried installing the sample app onto an older version of Raspbian and it works perfectly. Specifically, I used the 2018-06-27 release because I already had it on my computer. Thank you @thardbarger for bringing up this idea. @yuan0258 and @dhpp, I looked up instructions on installing PortAudio and the directions they gave went way over my head, but thanks for helping.
